repository/t.json

```diff
@@ -551,6 +551,18 @@
 				}
 			]
 		},
+		{
+			"name": "Template Tag",
+			"details": "https://github.com/ijlee2/sublime-syntax-definition-template-tag",
+			"description": "Provides syntax highlighting for <template> tags",
```

Comment:
You could leave this empty and manage it from the GitHub repo. It's ok if it says "sublime text" in the description.